.github/workflows/validate.yaml

```diff
@@ -35,9 +35,25 @@ jobs:
           sudo mv /usr/bin/yq /usr/local/bin/yq
 
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           path: ./src/github.com/${{ github.repository }}
+          fetch-depth: 0
```

Comment:
This is supposed to fetch history for all branches and tags (btw we exclude tags from fetching), is this required here?

Reply:
Yes, it was failing with `fatal: refusing to merge unrelated histories` without it, see https://github.com/openshift-knative/serverless-operator/actions/runs/3496276603/jobs/5853997426